core/src/apps/bitcoin/scripts.py

```diff
@@ -41,7 +42,11 @@ def input_derive_script(
     pubkey: bytes,
     signature: bytes,
 ) -> bytes:
-    if script_type == InputScriptType.SPENDADDRESS:
+    if script_type in (
```

Comment:
This should be something like this:

`if script_type == InputScriptType.SPENDADDRESS or coin.decred and script_type in [InputScriptType.SPENDSSRTX, InputScriptType.SPENDSSGEN]`